pallets/handles/src/handles_signed_extension.rs

```diff
@@ -0,0 +1,130 @@
+//! Substrate Signed Extension for validating requests to the handles pallet
```

Comment:
What about just calling the file extension.rs since we are already in the handles directory?

Reply:
For now, just being consistent with stateful storage.  This all likely going to change anyways as we have been accruing signed extensions.